src/components/enterprise-user-subsidy/data/hooks/hooks.js

```diff
@@ -244,3 +245,21 @@ export function useCustomerAgreementData(enterpriseId) {
 
   return [customerAgreement, isLoading];
 }
+
+const getRedeemablePoliciesData = async ({ queryKey }) => {
+  const enterpriseId = queryKey[1];
+  const userID = queryKey[2];
+  const response = await fetchRedeemableLearnerCreditPolicies(enterpriseId, userID);
+  return camelCaseObject(response.data);
+};
+
+export function useRedeemableLearnerCreditPolicies(enterpriseId, userID) {
+  const { data: redeemableLearnerCreditPolicies, isLoading, error } = useQuery({
+    queryKey: ['redeemablePolicies', enterpriseId, userID],
+    queryFn: getRedeemablePoliciesData,
+  });
+  if (error) {
+    logError(error);
```

Comment:
Note: function calls in component re-renders are executed on each re-render of a component. This means if the component rendering this custom hook re-renders but its underlying state and/or props remain constant, this `logError` call might be called more than desired (ideally, once, when the error happens).

If calling `logError` is a side effect of the `useQuery`, you might put this logic in a `useEffect` or in an `onError` argument to the `useQuery` call ([docs](https://tanstack.com/query/v4/docs/react/reference/useQuery)).

Reply:
I think the `onError` argument is close to deprecating soon. I'll move this logic in a `useEffect` block.